app/scripts/modules/core/src/deploymentStrategy/DeploymentStrategySelector.tsx

```diff
@@ -83,8 +83,15 @@ export class DeploymentStrategySelector extends React.Component<
     this.selectStrategy(this.props.command.strategy, true);
   }
 
+  public onChange = (key: string, value: any) => {
+    this.props.onFieldChange(key, value);
+
+    // This hack is needed to force the component to re-render when the angular deployment-strategy-selector updates the underlying command
+    this.setState({ AdditionalFieldsComponent: this.state.AdditionalFieldsComponent });
```

Comment:
Can we use forceUpdate? https://reactjs.org/docs/react-component.html#forceupdate
```suggestion

    // This hack is needed to force the component to re-render when the angular deployment-strategy-selector updates the underlying command
    this.forceUpdate();
```